src/libs/AxelarQueryAPI.ts

```diff
@@ -568,6 +568,37 @@ export class AxelarQueryAPI {
     }
   }
 
+  public async estimateITSFee(params: HopParams, options?: EstimateMultihopFeeOptions) {
+    // validate the source and destination chains
+    await throwIfInvalidChainIds([params.sourceChain, params.destinationChain], this.environment);
+
+    // TODO: Set this flag dynamically based on the source chain and destination chain
+    const isAmplifierRoute = true;
+
+    if (isAmplifierRoute) {
+      // estimate the gas fee for two hops for amplifier route
+      return this.estimateMultihopFee(
```

Comment:
all ITS transfers will go through ITS hub in the upcoming release (next ~2 weeks), so let's build towards that

Reply:
sounds good!